### PR TITLE
Ready

### PR DESCRIPTION
**Pull request by gmarcos87**
In this occasion I have managed to reduce the whole backend to just two packages, firstbootwizard and ubus-lime-fbw.

The most significant modifications are:
- Scan only in mesh or adhoc
- When applying the configuration do not delete the AP in wireless
- Add signal and bssid to file name lime-defaults
- Scan at startup if the router does not yet have a configuration selected

After accepting this pull request we can migrate these packages to lime-packages.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 env
+.history/
```
Comment (nicopace) [inline .gitignore]: This looks it is not related with the project

**Review by nicopace — COMMENTED** [review]
it looks ok.
haven't tested it

File: .gitignore (hunk 1 above)
Comment (gmarcos87) [reply .gitignore]: Done
